src/Life/Main/Remove.hs

```diff
@@ -26,7 +26,7 @@ lifeRemove lPath = whatIsLife >>= \case
     OnlyLife _ -> abortCmd "remove" "dotfiles/ directory doesn't exist"
     OnlyRepo _ -> abortCmd "remove" ".life file doesn't exist"
     -- actual life remove process
-    Both _ _ -> withSynced $ do
+    Both _ _ -> withSynced (Branch "master") $ do
```

Comment:
I see that `Branch "master"` will be used a lot, so it's nice to have a separate function for that and to import it from `Life.Github` module 🙂